codebase/superdarn/src.bin/tk/plot/map_plot.1.12/make_fov.c

```diff
@@ -103,3 +103,78 @@ struct PolygonData *make_fov(double tval,struct RadarNetwork *network,
 
     return ptr;
 }
+
+
+struct PolygonData *make_fov_data(struct GridData *gptr,struct RadarNetwork *network,
```

Comment:
Can you please add some comments on what this function does, parameters and what it returns?